extensions/database-documentation/installTypings.js

```diff
@@ -0,0 +1,17 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+var https = require('https');
```

Comment:
Remove this, not needed for extensions living in the ADS repo. The refs.d.ts will point directly to them instead.